src/App.php

```diff
@@ -133,13 +133,15 @@ public function run($request = null, $response = null)
     /**
      * Handle Route.
      *
-     * @param mixed $routeInfo
-     * @param PennyEventInterface $event
+     * @param RouteInfoInterface $routeInfo
```

Comment:
this is the internal method that handle RouteInfoInterface check, so param need accept not a RouteInfoInterface, so the RuntimeException will be executable!